libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/ScheduledMailDialogRenderer/RecipientsSelect/RecipientsSelectRenderer.tsx

```diff
@@ -347,8 +352,8 @@ export class RecipientsSelectRenderer extends React.PureComponent<IRecipientsSel
         return this.renderMultiValueItemContainer(email, removeIcon);
     };
 
-    private isExternalRecipientAddedByAuthor(recipient: IScheduleEmailExternalRecipient) {
```

Comment:
only the author can add external recipients. Here we check whether the new recipient was present already to distinguish the new ones. If the external recipient is new and the user is not the author an error bubble will be shown